Cython/Compiler/Nodes.py

```diff
@@ -2339,6 +2340,16 @@ def code_object(self):
         return self.py_func.code_object if self.py_func else None
 
     def analyse_declarations(self, env):
+        if self.decorators:
+            for decorator in self.decorators:
+                func = decorator.decorator
+                if func.is_name:
+                    if func.name == 'classmethod' or func.name == 'staticmethod':
+                        error(self.pos, "Cannot handle these decorators yet")
+                    if func.name == 'property':
```

Comment:
We should probably error on everything but this for now.

Reply:
errors on anything except `property` and `staticmethod`